prover/lang/kore-lang.md

```diff
@@ -140,12 +140,15 @@ only in this scenario*.
 ```k
   syntax Variable ::= VariableName "{" Sort "}" [klabel(sortedVariable)]
   syntax SetVariable ::= SharpName [klabel(setVariable)]
+  syntax Context ::= VariableName "[" Pattern "]" [klabel(context)]
```

Comment:
What does this represent? I thought a context is a pattern with a 'hole', which is a variable. So, does `VariableName` design the variable which plays the role of hole in the `Pattern`?

Reply:
Also, please explain the meaning of VariableName here.